_projects/13_iati_datastore.md

```diff
@@ -0,0 +1,7 @@
+---
+title: IATI Datastore
+tags:
+    - tools
+link: https://iatidatastore.iatistandard.org/ 
+
```

Comment:
Needs a `---` just here.

Reply:
where?

Reply:
Line 6. The frontmatter needs to be sandwiched in `---`s, [as shown in the documentation](https://jekyllrb.com/docs/front-matter/).